Samples/office-add-in-commands/auto-open-task-pane/README.md

```diff
@@ -0,0 +1,171 @@
+
+With the autoopen feature, you can explicitly define or allow the user to define whether a specific task pane add-in persists in a specific document.
+
+For more information and best practices on using this feature, see [Automatically open a task pane with a document](https://learn.microsoft.com/office/dev/add-ins/develop/automatically-open-a-task-pane-with-a-document).
+
+## Implementation
+
+### Specify the task pane to open
+
+To specify the task pane to open automatically, the sample sets the **TaskpaneId** value to **Office.AutoShowTaskpaneWithDocument** in the manifest file. You can only set this value on one task pane. If you set this value on multiple task panes, the first occurrence of the value will be recognized and the others will be ignored.
+
+```xml
+<Action xsi:type="ShowTaskpane">
+  <TaskpaneId>Office.AutoShowTaskpaneWithDocument</TaskpaneId>
+  <SourceLocation resid="Contoso.Taskpane.Url" />
+</Action>
+```
+
+### Tag the document to automatically open the task pane
+
+The sample uses the Office JS **settings.set** method to set **Office.AutoShowTaskpaneWithDocument** to **true** as shown in the following code from the **home.js** file. To turn autooppen off, the code calls **settings.remove** to remove the setting.
+
+```javascript
+function setAutoOpenOn() {
+    Office.context.document.settings.set(
+        'Office.AutoShowTaskpaneWithDocument',
+        true
+    );
+    Office.context.document.settings.saveAsync();
+    showNotification(
+        'The auto-open setting has been set to ON on this document'
+    );
+}
+
+function setAutoOpenOff() {
+    Office.context.document.settings.remove(
+        'Office.AutoShowTaskpaneWithDocument'
+    );
+    Office.context.document.settings.saveAsync();
+    showNotification(
+        'The auto-open setting has been set to OFF on this document'
+    );
+}
+```
+
+> Note: You can also modify the autoopen setting by using OpenXML. For more information, see [Automatically open a task pane with a document](https://learn.microsoft.com/office/dev/add-ins/develop/automatically-open-a-task-pane-with-a-document).
+
+## Add-in installation requirement
+
+It is important to highlight that the **pane that you designate will only automatically open IF** , by the time the user opens the document, your **add-in is already installed on the users device**. If users open a document and they do not have your add-in already installed then nothing will happen, the setting will be ignored.
+
+If you need to distribute the add-in with the document, so that users are prompted to install it, you also need to set the pane visibility property to 1. You can only do this via OpenXML.
```

Comment:
```suggestion
If you need to distribute the add-in with the document, so that users are prompted to install it, you also need to set the pane visibility property to `1`. You can only do this via OpenXML.
```